### ISSUE TITLE
Add minimal client for mips/mipsel/arm platform

### ISSUE THREAD

**Issue by DatanoiseTV** [schollz/find3#42]
Many embedded wireless platforms have enough resources to run a few tasks in the background. By building a minimal, single binary stand-alone version for MIPS and ARM, wireless routers could be used to gain passive and active data.

**Comment by schollz** [comment]
@DatanoiseTV Which minimal binary are you referring to? The CLI scanner?

**Comment by DatanoiseTV** [comment]
@schollz Yes, the CLI scanner.

**Comment by schollz** [comment]
Yeah, so that can easily be built for ARM since Go cross compiles easily. The problem though is how to collect the Bluetooth/WiFi signals. In the scanner I'm using Linux-specific utilities (bluez and iw utilities). I think that these would depend on the specific device, unless there is a C-library maybe I can use to do this generically?

**Comment by DatanoiseTV** [comment]
@schollz If you use the same API in Go which "iwlist scan" uses for scanning, you should have no problems to just use the Linux WiFi scanning API.
